packages/fast-components-styles-msft/src/utilities/elevation.spec.ts

```diff
@@ -0,0 +1,63 @@
+import {
+    ambientShadowConfig,
+    applyElevation,
+    ElevationMultiplier,
+    elevationShadow,
+} from "./elevation";
+import defaultDesignSystem, { DesignSystemResolver } from "../design-system";
+
+describe("elevationShadow", () => {
+    test("should calculate elevation value correctly", () => {
+        expect(
+            elevationShadow(ElevationMultiplier.e1, "#000", ambientShadowConfig)(
+                defaultDesignSystem
+            )
+        ).toBe("0px 0.1px 0.225px rgba(0,0,0,0.11)");
+        expect(
+            elevationShadow(ElevationMultiplier.e4, "#000", ambientShadowConfig)(
+                defaultDesignSystem
+            )
+        ).toBe("0px 0.3px 0.9px rgba(0,0,0,0.11)");
+        expect(
+            elevationShadow(ElevationMultiplier.e13, "#000", ambientShadowConfig)(
+                defaultDesignSystem
+            )
+        ).toBe("0px 3.6px 10.8px rgba(0,0,0,0.18)");
+        expect(
+            elevationShadow(ElevationMultiplier.e17, "#000", ambientShadowConfig)(
+                defaultDesignSystem
+            )
+        ).toBe("0px 14.4px 43.2px rgba(0,0,0,0.18)");
+    });
+});
+
+describe("applyElevation", () => {
+    test("should calculate elevation value correctly", () => {
```

Comment:
These tests all say the same thing, if one fails we need to know which so I would recommend you add in something to distinguish each one.